prow/jobs/kyma/kyma-compass-integration.yaml

```diff
@@ -49,54 +47,58 @@ presubmits: # runs on PRs
             - name: CLOUDSDK_COMPUTE_ZONE
               value: "europe-west4-b"
             - name: INPUT_CLUSTER_NAME
-              value: "compass-integration-periodic-on-demand"
+              value: "compass-integration"
```

Comment:
I think we should use randomly generated cluster names, similarly to how it's done in `kyma-gke-integration.sh`. It wasn't a problem before because periodics and on-demands didn't overlap.